homeworks/Andrii.Morozov_AndriiMorozov88/10-components/tik-tak-toe/src/app/app.component.html

```diff
@@ -0,0 +1,11 @@
+<header class="header">
+    <app-icon [iconSize]="iconSize" [icon]="icon"></app-icon>
```

Comment:
Просто size. компонент називається icon тому не потрібно на початку додавати ще раз icon
```suggestion
    <app-icon [size]="iconSize" [icon]="icon"></app-icon>
```